extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetS3SimpleMinIOTest.java

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table;
+
+import io.deephaven.extensions.s3.S3Instructions.Builder;
+import io.deephaven.extensions.s3.testlib.SingletonContainers.MinIO;
+import io.deephaven.extensions.s3.testlib.SingletonContainers;
+import io.deephaven.stats.util.OSUtil;
+import org.junit.BeforeClass;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+
+import static org.junit.Assert.assertFalse;
+
+public class ParquetS3SimpleMinIOTest extends ParquetS3SimpleTestBase {
+
+    @BeforeClass
+    public static void initContainer() {
+        // TODO(deephaven-core#5116): MinIO testcontainers does not work on OS X
+        assertFalse("OSUtil.runningMacOS()", OSUtil.runningMacOS());
```

Comment:
This needs to be an _assumption_ not an _assertion_. `org.junit.Assume`

Reply:
Well, I _think_ it should be `assumeTrue` - unconditionally causing unit test failure for a specific platform is not ideal; the idea is that users who do something like `./gradlew check` on OSX should _not_ fail because of the MinIO test (the tests should essentially be skipped). To more specifically test this out on OSX, ensure that `./gradlew extensions-parquet-table:testOutOfBand` completes successfully.

Reply:
Actually, I think you are right in using `assumeFalse`.